pytest_httpserver/httpserver.py

```diff
@@ -675,6 +724,61 @@ def dispatch(self, request: Request) -> Response:
 
         return response
 
+    def _waiting_result_set(self, value: bool) -> None:
```

Comment:
Could this be named `_set_waiting_result`? I think having a method starting with a verb sounds better.